src/controller/CHIPDeviceController.h

```diff
@@ -123,6 +121,24 @@ struct CommissionerInitParams : public ControllerInitParams
     OperationalCredentialsDelegate * operationalCredentialsDelegate = nullptr;
 };
 
+/**
+ * @brief
+ * Used for make current OnSuccessCallback & OnFailureCallback works when interaction model landed, it will be removed
+ * in the future.
+ */
```

Comment:
Again, I'm not sure what the `future` is. If you open an issue # we can discuss it there :)
Maybe your comment is perfectly right, but for my part I don't know how you think it is supposed to work.

For example, I can imagine that `CHIPClientCallbacks` can be completely changed in order to implement the `InteractionModelDelegate` interface before beeing passed as an init param. But instead of comments referring to `future` it would makes more sense to have an issue number that explains what is the plan.

Reply:
Done